### PR TITLE
added a 4rth file

### PR DESCRIPTION
**Pull request by marco-giuliani**
added a fourth

### CODE REVIEW

File: file4.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+This is another file
```
Comment (adeydev) [inline file4.txt]: no full stop. sort it.

**Review by adeydev — CHANGES_REQUESTED** [review]
not happy

File: file4.txt (hunk 1 above)
Comment (marco-giuliani) [reply file4.txt]: made the change boss

**Comment by marco-giuliani** [comment]
Reply to email

File: file4.txt (hunk 1 above)
Comment (adeydev) [reply file4.txt]: right, ok

**Review by adeydev — APPROVED** [review]

**Pull request merged**